test/parallel/test-repl.js

```diff
@@ -818,7 +818,41 @@ const tcpTests = [
       kArrow,
       '',
       'Uncaught:',
-      /^SyntaxError: .* dynamic import/,
+      'SyntaxError: Cannot use import statement inside the Node.js REPL, \
+alternatively use dynamic import: const comeOn = await import(\'fhqwhgads\');',
+    ]
+  },
+  {
+    send: 'import { export1, export2 } from "module-name"',
```

Comment:
Can you add a test case for `import { export1 as localName1, export2 } from …` please?